### PR TITLE
Create AWS local development tips

### PR DESCRIPTION
**Pull request by nbon12**
Creating helpful documentation for newcomers who wish to inject AWS data into their local koku.
Addresses #1574

### CODE REVIEW

File: docs/source/aws.rst [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+1)  Create a Provider
+
+Send a POST request to
+
+`http://<your_host>:8000/api/cost-management/v1/providers/`
+
+With the content:
+
+Provider Resource Name:arn:aws:iam::589173575009:role/CostManagement
```
Comment (adberglund) [inline docs/source/aws.rst]: This shouldn't be public.

File: docs/source/aws.rst [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+1)  Create a Provider
+
+Send a POST request to
+
+`http://<your_host>:8000/api/cost-management/v1/providers/`
+
+With the content:
+
+Provider Resource Name: <whatever your arn is>
```
Comment (werwty) [inline docs/source/aws.rst]: Might be worth it to have an example call here:
```
http POST 0.0.0.0:8000/api/cost-management/v1/providers/ name='AWSProvider' type='AWS' \
authentication:='{"uuid": "e6d90e5a-feb9-472e-97f3-b96839206f4f","provider_resource_name": "arn:aws:iam::01234567890:my_aws_role"}'
```

File: docs/source/aws.rst [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+Send a GET request to
+`http://<your_host>:5000/api/cost-management/v1/download/`
+That _should_ trigger the worker to load the data into the DB. 
+
+You may want to check the logs to see if the loading is occuring:
+
+`docker-compose logs -f koku-server koku-worker`
+
+
+koku_beat is the Celery scheduler: Periodic Tasks — Celery 4.3.0 documentation
```
Comment (werwty) [inline docs/source/aws.rst]: Did you mean to add link here to the celery docs?

Comment (nbon12) [reply docs/source/aws.rst]: no, it was more of a random piece of information that I wasn't sure where to put.

File: docs/source/aws.rst (hunk 2 above)
Comment (nbon12) [reply docs/source/aws.rst]: @werwty I added this now!

File: docs/source/aws.rst [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+Send a POST request to
+
+`http://<your_host>:8000/api/cost-management/v1/providers/`
+
+With the content:
+
+Provider Resource Name: <whatever your arn is>
+For example: 
+http POST 0.0.0.0:8000/api/cost-management/v1/providers/ name='AWSProvider' type='AWS' \
```
Comment (werwty) [inline docs/source/aws.rst]: Can you put this into an RST codeblock?

File: docs/source/aws.rst [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+
+You'll need to set those environment variables with your AWS access and secret key. And then you can call docker-compose up
+
+=========================
+How to Create a Provider
+=========================
+Send a POST request to
+`http://<your_host>:8000/api/cost-management/v1/providers/`
```
Comment (infinitewarp) [inline docs/source/aws.rst]: For RST inline literals like this, you want double backticks. See: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup

Alternatively, if you want block code, you'll need it indented by four spaces with a blank line _both_ before and after in. See: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#literal-blocks


```suggestion
```

File: docs/source/aws.rst [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+
+You'll need to set those environment variables with your AWS access and secret key. And then you can call docker-compose up
+
+=========================
+How to Create a Provider
+=========================
+Send a POST request to
```
Comment (infinitewarp) [inline docs/source/aws.rst]:
```suggestion
Send a POST request to ``http://<your_host>:8000/api/cost-management/v1/providers/``
```

File: docs/source/aws.rst [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+`http://<your_host>:8000/api/cost-management/v1/providers/`
+With the content:
+Provider Resource Name: <whatever your arn is> 
+ 
+  http POST 0.0.0.0:8000/api/cost-management/v1/providers/ name='AWSProvider' type='AWS' \ authentication:='{"uuid": "e6d90e5a-feb9-472e-97f3-b96839206f4f","provider_resource_name": "arn:aws:iam::01234567890:my_aws_role"}'
```
Comment (infinitewarp) [inline docs/source/aws.rst]:
```suggestion
    http POST 0.0.0.0:8000/api/cost-management/v1/providers/ name='AWSProvider' type='AWS' \ authentication:='{"uuid": "e6d90e5a-feb9-472e-97f3-b96839206f4f","provider_resource_name": "arn:aws:iam::01234567890:my_aws_role"}' 
```

File: docs/source/aws.rst [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+ 
+  http POST 0.0.0.0:8000/api/cost-management/v1/providers/ name='AWSProvider' type='AWS' \ authentication:='{"uuid": "e6d90e5a-feb9-472e-97f3-b96839206f4f","provider_resource_name": "arn:aws:iam::01234567890:my_aws_role"}' 
+ 
+You should receive a 200 range response confirming that the Provider was created.
+
+=============
+Trigger Masu
+=============
+
+Send a GET request to `http://<your_host>:5000/api/cost-management/v1/download/` That _should_ trigger the worker to load the data into the DB.
```
Comment (infinitewarp) [inline docs/source/aws.rst]: Italics in RST use single asterisks.

```suggestion
Send a GET request to ``http://<your_host>:5000/api/cost-management/v1/download/`` That *should* trigger the worker to load the data into the DB.
```

File: docs/source/aws.rst [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+
+You may want to check the logs to see if the loading is occuring:
+
+`docker-compose logs -f koku-server koku-worker`
```
Comment (infinitewarp) [inline docs/source/aws.rst]:
```suggestion
    docker-compose logs -f koku-server koku-worker
```

File: docs/source/aws.rst [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+
+If you see this error:
+`Provider skipped: da56f23c-100e-4158-8544-0a146a9dc64d Valid: True Backing off: True`
```
Comment (infinitewarp) [inline docs/source/aws.rst]:
```suggestion

    Provider skipped: da56f23c-100e-4158-8544-0a146a9dc64d Valid: True Backing off: True

```

**Pull request merged**